src/ctl.c

```diff
@@ -543,6 +544,7 @@ static const ctl_named_node_t stats_arenas_i_node[] = {
 	{NAME("metadata_thp"),	CTL(stats_arenas_i_metadata_thp)},
 	{NAME("tcache_bytes"),	CTL(stats_arenas_i_tcache_bytes)},
 	{NAME("resident"),	CTL(stats_arenas_i_resident)},
+	{NAME("dropped_vm"),	CTL(stats_arenas_i_dropped_vm)},
```

Comment:
@davidtgoldblatt -- as we discussed a while ago, adding the counter to track the extents_leak (which also got renamed to extents_drop_vm in this diff), but under a different name. Does this sound okay / any suggestions?

Reply:
Sounds good ("abandoned" or "orphaned" would work too if you like one of those better). Saying "leak" in user-visible stats is a recipe for confusion imo.

Reply:
I think "abandoned" sounds better. Making the switch now.